src/components/widgets/camera/CameraItem.vue

```diff
@@ -123,6 +123,7 @@ export default class CameraItem extends Vue {
         const hostUrl = new URL(document.URL)
         const url = new URL(this.cameraUrl, hostUrl.origin)
         url.searchParams.set('cacheBust', this.refresh.toString())
+        this.request_start_time = performance.now()
```

Comment:
What is this for?

Reply:
it's a fix for the broken frame load scheduler. without it the delay between frames will eventually be 0 so it's loading a frame on every tick instead of when it's actually scheduled to load